hubblestack_nova/command.py

```diff
@@ -132,6 +132,7 @@ def audit(data_list, tags, debug=False):
 
                         found = False
                         if cmd_ret:
+                            tag_data['output'] = cmd_ret
```

Comment:
Looks like you accidentally included your change from https://github.com/hubblestack/hubble-salt/pull/78. Could you remove it so we can manage the changes independently?